xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp

```diff
@@ -1400,6 +1400,12 @@ bool CRenderManager::GetStats(int &lateframes, double &pts, int &queued, int &di
 
 void CRenderManager::CheckEnableClockSync()
 {
+  if (CSettings::GetInstance().GetBool(CSettings::SETTING_VIDEOPLAYER_USEDISPLAYASCLOCK))
+  {
+    m_clockSync.m_enabled = true;
```

Comment:
@FernetMenta 

provides the gap/error calc for passthrough, but also it shifts the dvd audio clock which provides the callbacks for getting streams clock (if I remember correctly).

basically when playing for instance 23.976 on screen at 60, rr gets quite erratic, without getting stable. allowing that offset calculation rr gets stable quite fast. if you give me any other pointer I will gladly go again with research...